L/LLVM/common.jl

```diff
@@ -20,6 +20,7 @@ const llvm_tags = Dict(
     v"16.0.6" => "499f87882a4ba1837ec12a280478cf4cb0d2753d", # julia-16.0.6-2
     v"17.0.6" => "0007e48608221f440dce2ea0d3e4f561fc10d3c6", # julia-17.0.6-5
     v"18.1.7" => "ed30d043a240d06bb6e010a41086e75713156f4f", # julia-18.1.7-2
+    v"19.0.0" => "6c8f4c12da7b3ac3d45e9fc9f68e2da7f559cf76", # julia-release/19.x
```

Comment:
Reminder: we need to create the git tag on the commit we will eventually use here.

Reply:
For the record, git tag created: https://github.com/JuliaLang/llvm-project/releases/tag/julia-19.1.1-0